test/mlir/conversion/onnx_to_torch/flatten/flatten.onnx.mlir

```diff
@@ -2,8 +2,8 @@
 module {
   func.func @main_graph(%arg0: tensor<1x1x160x160xf32>) -> tensor<1x25600xf32> attributes {input_names = ["0"], output_names = ["1"]} {
     %0 = "onnx.Flatten"(%arg0) {axis = 1 : si64, onnx_node_name = "Flatten_0"} : (tensor<1x1x160x160xf32>) -> tensor<1x25600xf32>
-//CHECK: %[[ENDDIM:.*]] = torch.constant.int 3
 //CHECK: %[[DIM:.*]] = torch.constant.int 1
+//CHECK: %[[ENDDIM:.*]] = torch.constant.int 3
```

Comment:
Maybe https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-dag-directive is what we need where the order is not important?